lib/src/main/java/com/diffplug/spotless/kotlin/KtLintStep.java

```diff
@@ -43,18 +44,22 @@ public static FormatterStep create(Provisioner provisioner) {
 	}
 
 	public static FormatterStep create(String version, Provisioner provisioner) {
-		return create(version, provisioner, false);
+		return create(version, provisioner, Collections.emptyMap());
 	}
 
-	public static FormatterStep createForScript(String version, Provisioner provisioner) {
-		return create(version, provisioner, true);
+	public static FormatterStep create(String version, Provisioner provisioner, Map<String, String> userData) {
+		return create(version, provisioner, false, userData);
```

Comment:
`createForScript(String version, Provisioner provisioner)` is still missing.

Reply:
oops. added it back!